src/Components/Notifications/PartnerOfferCreatedNotification.tsx

```diff
@@ -32,7 +32,12 @@ export const PartnerOfferCreatedNotification: FC<PartnerOfferCreatedNotification
   const partnerOffer = item?.partnerOffer
   const artwork = extractNodes(offerArtworksConnection)[0]
   const { hasEnded } = useTimer(partnerOffer?.endAt || "")
-  let subtitle = "Review the offer on your saved artwork"
+  const isOfferFromSaves = partnerOffer?.source === "SAVE"
+
+  let subtitle
+  if (isOfferFromSaves) subtitle = "Review the offer on your saved artwork"
+  else subtitle = "Review the offer before it expires"
```

Comment:
**Suggestion (non-blocking):** Maybe we could use the ternary operation in defining the `subtitle`?
```suggestion
  let subtitle = isOfferFromSaves ? "Review the offer on your saved artwork" : "Review the offer before it expires"
```